src/helpers/prettyDate.ts

```diff
@@ -11,5 +11,10 @@ export const prettyDate = (dateString: string): string => {
   const day = new Date(dateString);
   const dayName = days[day.getDay()];
   const date = new Date(dateString).toLocaleDateString();
-  return `${dayName}, ${date}`;
+  const hours = new Date(dateString).getHours();
+  let minutes: string | number = new Date(dateString).getMinutes();
+  minutes < 10 ? (minutes = "0" + minutes) : (minutes = minutes);
```

Comment:
what is happening in this line?

Reply:
if minutes value is less then 10 time would be retuned as `11:3` instead of desired `11:03`